.github/workflows/ci.yml

```
@@ -97,10 +97,13 @@ jobs:
         include:
           - name: ubuntu-20.04
             runs-on: ubuntu-20.04
+            driver-options: -DIREE_HAL_DRIVER_CUDA=ON -DIREE_HAL_DRIVER_HIP=ON
           - name: windows-2022
             runs-on: windows-2022
+            driver-options: -DIREE_HAL_DRIVER_CUDA=ON -DIREE_HAL_DRIVER_HIP=ON
           - name: macos-14
             runs-on: macos-14
+            driver-options: -DIREE_HAL_DRIVER_METAL=ON -DIREE_HAL_DRIVER_VULKAN=OFF
```

Edit: actually, reading https://github.com/iree-org/iree/pull/17564#issuecomment-2158905858 from @benvanik  again
> I think regardless of choice we need to not tie defaults to what the CI/bots/packaging systems do - those should never use defaults. The big red flag here for me is that changing a source build default (the only thing the defaults should be used for) is being used to control packaging. The only bots that should use defaults are smoketest bots for user flows ("if a user checks out a build with no settings, can we build?").

Let's keep this, but then maybe also add `-DIREE_HAL_DRIVER_VULKAN=ON` to the others instead.

---

(Original comment)

Oh, I think this should be safe to set to just
```suggestion
          - name: macos-14
            runs-on: macos-14
            driver-options: -DIREE_HAL_DRIVER_METAL=ON
```

Vulkan should default to off on macOS. See also https://github.com/iree-org/iree/commit/e7a18982c3c135aa35cf5652cf2c7d8c071b93d7